napari/layers/surface/surface.py

```diff
@@ -123,7 +123,7 @@ def __init__(
             visible=visible,
         )
 
-        self.events.add(interpolation=Event, rendering=Event)
```

Comment:
These events seems deprecated, I didn't find reference of it anywhere, if needed I can reinstate them

Reply:
hmm looks like you're right, I don't see them being called anywhere so maybe we can drop for now. We don't support multiple interpolation modes or rendering modes for surface right now anyway. maybe was a mistake that they are in there